docs/contributing/testing.md

```diff
@@ -65,27 +65,36 @@ You should test that your contribution works:
 
 ## 5. Run the automated tests
 
+Automated testing helps ensure that our code is as free of bugs as it can be. It also lets us know immediately if a change we make breaks any other part of the code.
+
 In your project folder, run `npm test` to run the automated tests, including linting.
 
 If a test fails, you should check your code for any errors, then update any tests you need to.
 
 ## 6. Write new tests
 
+You can [read more about the different types of tests in this project](../releasing/testing-and-linting).
+
 You should write new tests if you’ve created a new component, or changed the way a component works by:
 
 - changing or adding to the component's JavaScript code
 - changing or adding to the component's Nunjucks macro
 - creating or updating a Sass mixin or function
 
-Test files use examples from each component’s `.yaml` file, for example `src/govuk/components/button/button.yaml`. When you add or update tests, you can use the existing examples or add new ones.
+If you're new to testing, see existing test files for examples of things to do. Don't let the tests keep you from submitting your contribution! If you're not sure how to do this or are having trouble, submit your pull request anyway. We will help you create the tests and sort out any kind of problem during code review.
```

Comment:
Some minor things:
- Change 'Don't' to 'Do not'  
- "If you're not sure how to do this..." Is 'this' how to do the pull request or the testing?  
- Alternative text tweak for the last line - 'We will help you create the tests and solve problems during code review.'